connaisseur/validators/cosign/cosign_validator.py

```diff
@@ -19,23 +19,25 @@ class CosignValidator(ValidatorInterface):
     name: str
     keys: list
 
-    def __init__(self, name: str, pub_keys: list, **kwargs):
+    def __init__(self, name: str, trust_roots: list, **kwargs):
         self.name = name
-        self.keys = pub_keys
+        self.keys = trust_roots
```

Comment:
```suggestion
        self.trust_roots = trust_roots
```

and then adopt in the following lines